docs/conf.py

```diff
@@ -78,6 +78,9 @@
     ('py:exc', 'BadOperationException'),
     ('py:exc', 'Exception'),
     ('py:exc', 'ValueError'),
+    ('py:class', 'integer -- return number of occurrences of value'),
+    ('py:class', 'integer -- return first index of value.'),
```

Comment:
What are these two exceptions for?

Reply:
`('py:class', 'integer -- return number of occurrences of value'),  ` is for the following complaint: 

`/usr/lib/python3.6/collections/__init__.py:docstring of mod_pywebsocket.stream.deque.count::py:class reference target not found: integer -- return number of occurrences of value`

`('py:class', 'integer -- return first index of value.'),` is for this:

`/usr/lib/python3.6/collections/__init__.py:docstring of mod_pywebsocket.stream.deque.index::py:class reference target not found: integer -- return first index of value.`

`stream.py` calls `from collections import deque`, these two docstrings are in [collections.abc](https://docs.python.org/3/library/collections.abc.html). Somehow, the complaints we got above are related to the version of Sphinx used. in [collections.abc](https://docs.python.org/3/library/collections.abc.html), it used version 2.4.4. When replacing the sphinx version used in this PR from 3.3.1 to 2.4.4, we wouldn't get these complaints. I had a look on sphinx extensions etc. but couldn't find any solutions. Would it make sense to change sphinx version to 2.4.4 to be in line with the version python uses?

Reply:
It seems that some work went into fixing pydocs with Sphinx 3 in Python 3.8+ (https://bugs.python.org/issue40204), but not in 3.6 or 3.7, so let's use Sphinx 2 instead.